### PR TITLE
fix radiator deltatime

### PR DESCRIPTION
**Pull request by Ilya246**
## About the PR
fixes radiators improperly accounting for deltatime
successfully built locally and tested ingame

**Changelog**

:cl:
- tweak: Increase radiator heat transfer rate.

### CODE REVIEW

**Comment by deltanedas** [comment]
may as well replace the EntityManager calls with proxy

File: Content.Server/Atmos/EntitySystems/HeatExchangerSystem.cs [hunk 1]
```diff
@@ -47,6 +48,7 @@ private void CacheTileLoss(float val)
     private void OnAtmosUpdate(EntityUid uid, HeatExchangerComponent comp, AtmosDeviceUpdateEvent args)
     {
         if (!EntityManager.TryGetComponent(uid, out NodeContainerComponent? nodeContainer)
+                || !EntityManager.TryGetComponent(uid, out AtmosDeviceComponent? device)
```
Comment (deltanedas) [inline Content.Server/Atmos/EntitySystems/HeatExchangerSystem.cs:51]: TryComp and replace the others in the file

File: Content.Server/Atmos/EntitySystems/HeatExchangerSystem.cs [hunk 2]
```diff
@@ -21,6 +21,7 @@ public sealed class HeatExchangerSystem : EntitySystem
     [Dependency] private readonly AtmosphereSystem _atmosphereSystem = default!;
     [Dependency] private readonly IConfigurationManager _cfg = default!;
     [Dependency] private readonly NodeContainerSystem _nodeContainer = default!;
+    [Dependency] private IGameTiming _gameTiming = default!;
```
Comment (deltanedas) [inline Content.Server/Atmos/EntitySystems/HeatExchangerSystem.cs:24]: _timing is standard

Comment (Ilya246) [reply Content.Server/Atmos/EntitySystems/HeatExchangerSystem.cs:24]: atmos code uses _gameTiming

**Review by deltanedas — CHANGES_REQUESTED** [review]
a

File: Content.Server/Atmos/EntitySystems/HeatExchangerSystem.cs (hunk 2 above)
Comment (deltanedas) [reply Content.Server/Atmos/EntitySystems/HeatExchangerSystem.cs:24]: it also uses EntityManager.X everywhere doesnt mean it good

**Comment by deltanedas** [comment]
:trollface:

**Pull request merged**